### ISSUE TITLE
Second time lapse image is acquired too soon after the first one

### ISSUE THREAD

**Issue by EtalumaSupport** [EtalumaSupport/LumaViewPro#239]
If the z position is far away from the first Step in a Protocol, the time it takes to get to the correct focus is added to the first image start time and thus the second image comes too soon afterwards.  The first image "clock" starts with the initiation of the moving to the right focus, not when the first image is acquired.  This is most likely also occurring when a microplate Protocol "parks" at the last well imaged instead of the next to be imaged well between intervals.  In this case the initiation of the XY translation will be the t=0 rather than the first image acquisition.  So the first well of a multi-well Protocol is slightly different in timing, not important if the interval is 8 hours but at 6 seconds it's very noticeable.

**Comment by EtalumaSupport** [comment]
Issue #218 also introduces a timing error for being out of position before AND in between Intervals.

**Comment by ericweiner** [comment]
## Prior art (2026-07-23 doc cross-reference)

**Governing doc:** `FABLEAUDIT_PROTOCOL_STATE_2026-06-11.md` (PS-8, lines 18 & 97) -- `_start_t = datetime.now()` set at SCAN START (`sequenced_capture_runner:232`), inter-scan wait is `now()-_start_t < period` (`protocol_run_loop:113-121`). Fix `fe31ac8a` moved pacing to `time.monotonic()` but explicitly "overrun-drift semantics unchanged." The issue's exact complaint -- t=0 anchored at motion start, not first-image acquisition -- is the `_start_t`-at-scan-start mechanism PS-8 documents and the fix did NOT re-anchor.
**What this means:** GOVERNS / PARTIAL -- subsystem analyzed and half-fixed; the clock source was reworked but the specific first-interval anchor-point defect survives the monotonic migration.
**Cluster:** #238, #260 (protocol timing/UX).

Read the governing doc before re-analyzing this issue.

**Comment by ericweiner** [comment]
## Fix drafted on branch (pending merge)

Drafted on `fix/small-batch-2026-07` at `07f1df51` -- not merged; lands on `4.0.0-beta` only after the #747 bench gate passes and the batch suite gate is green.

**Root cause:** the timelapse pacing anchor had two writers bound to different events. Scans 2..N anchor at gate-open (spacing exactly = period), but scan 1's anchor was run() entry -- before run setup, the initial stage move, and step-0 autofocus -- so the first interval came up short by that lead-in (toward zero at a 6 s period).

**Fix (per the recorded decision):** the anchor is ESTABLISHED at the run's first completed acquisition, then MAINTAINED at gate-open for scans 2..N unchanged -- fixed cadence and overrun-drift semantics preserved; image count unaffected (run end is count-based). `modules/protocol_run_loop.py` + `modules/protocol_step_runner.py` + `modules/sequenced_capture_runner.py`.

**Regression test:** `tests/test_issue_239_first_interval_anchor.py` (simulated clock; first-interval red at the pre-fix SHA: 75.1 s < 100 s).

**Test plan to confirm fix:** 6 s period with Z parked far from step 0 -- file timestamps show image-1 -> image-2 spacing >= the period; a 3-interval run confirms later spacing unchanged.

Status: code-side drafted on branch; hardware verification pending -- please test after the branch merges.

**Comment by ericweiner** [comment]
## Ready for bench verification

Fix shipped on `4.0.0-beta` at `07f1df51` (merged in `ad901a62`); current shipping tip `19a6b513`. Scan 1 now anchors its period at first acquisition.

Bench-verified 2026-07-27 on an LS850 with a 6 s period: gate-to-gate intervals 6.49/5.99/6.00 s and 6.64/5.92 s -- the too-soon second image is gone (first interval = period + one-time frame-drain, then locked at 6.0 s).

Chris, please build the latest beta and verify with your timelapse, then either:
- **Close this issue** if resolved, OR
- **Add logs / a note** describing what is still happening.

**Comment by ericweiner** [comment]
## Census verification 2026-08-06 (pinned LVP f055615a)
Fix `07f1df51` (2026-07-24, bench-verified per its trail) postdates every field comment (latest 2023-09-03). Second-lapse-too-soon timing verified resolved at the pin. Disposition: Fixed, needs reporter verification on current beta.

**Comment by ericweiner** [comment]
## Pattern-pass disposition: **SINGLETON** (2026-09-03, pin `47a98931` = beta32)

A cross-issue pattern pass grouped the 124 open issues carrying an analysis by **shared root cause** (never shared symptom), then an adversarial pass attacked the grouping. **This issue has no sibling in that corpus** -- it stands on its own root. 77 of 124 came out this way; that is the expected shape, and no group was manufactured to reduce it.

**The root as it stands at the pin:** Timelapse pacing anchor had two writers; fix drafted on a branch, not merged -- gated on #747's bench pass.


**No fix started, no plan approved, no status change.** Triage ends at the root cause. Every annotation, including this one, is a hypothesis until checked against the code.

**Comment by EtalumaSupport** [comment]
Protocol now correctly returns to the start, ready to begin the subsequent interval.

<img width="585" height="1266" alt="Image" src="https://github.com/user-attachments/assets/fb9f35fa-3558-40f8-8a59-e36850b85cd6" />
